ui/app/src/components/CreateSiteDialog/CreateSiteDialogContainer.tsx

```diff
@@ -461,6 +470,11 @@ export function CreateSiteDialogContainer(props: CreateSiteDialogContainerProps)
     setDialog({ inProgress: false });
   }
 
+  function onBackgroundCreateClick() {
+    siteCreateSubscription.current?.unsubscribe();
+    handleClose();
+  }
+
```

Comment:
We can remove all together. Call handleClose directly on the button and request is cancelled by the effect.